garak/probes/phrasing.py

```diff
@@ -0,0 +1,111 @@
+"""Past Tense
+
+    Refusal training is widely used to prevent LLMs from generating harmful, undesirable, or illegal
+outputs. We reveal a curious generalization gap in the current refusal training approaches:
+simply reformulating a harmful request in the past tense  is often sufficient to jailbreak many state-
+of-the-art LLMs
+"""
+
+import random
+
+from garak import _config
+from garak.data import path as data_path
+from garak.probes.base import Probe
+
+
+class TenseMini:
+    def _minify_prompts(self):
+        random.shuffle(self.prompts)
```

Comment:
Should we consider providing a seed to ensure a reproducible shuffle?

```suggestion
        random.Random(self.seed).shuffle(self.prompts)
```

Creating a custom `Random` object avoids impacts to the global random generator, but provides a method to enable users create reproducibility when required and adds consistency between runs.

It is reasonable to defer this and as optional as the `seed` would need to come from some default or logged value overridable by configuration possibly injected from `_config.run.seed` by cli once the refactor to remove direct access extract access to that value from plugins occurs.

Reply:
The set for `random` will create consistency if no other probe or task accessing random occurs in a different order between runs, the example limits to a single probe. Since probes are instantiated in series two runs that used a dynamic probe and `phrasing` could cause the global random object to reach a different value at shuffle for separate runs.

This idea was optional so will land and we can circle back if we see this needs the consistency.